lib/msf/http/wordpress.rb

```diff
@@ -11,6 +11,7 @@ module Wordpress
       require 'msf/http/wordpress/uris'
       require 'msf/http/wordpress/users'
       require 'msf/http/wordpress/version'
+      require 'msf/http/wordpress/xmlrpc'
```

Comment:
I'm going to make this file name to ```msf/http/wordpress/xml_rpc``` since the mixin namespace is ```Msf::HTTP::Wordpress::XmlRpc```. I'm going to do it by myself when landing, no worry :)